util/stmtsummary/statement_summary.go

```diff
@@ -765,6 +775,15 @@ func (ssElement *stmtSummaryByDigestElement) add(sei *StmtExecInfo, intervalSeco
 		ssElement.execRetryCount += sei.ExecRetryCount
 		ssElement.execRetryTime += sei.ExecRetryTime
 	}
+	ssElement.sumKVTotal += sei.KVTotal
+	ssElement.sumPDTotal += sei.PDTotal
+	ssElement.sumBackoffTotal += sei.BackoffTotal
+	ssElement.sumWriteSQLRespTotal += sei.WriteSQLRespTotal
+	if sei.Prepared {
+		ssElement.prepared = true
+	} else {
+		ssElement.prepared = false
+	}
```

Comment:
`ssElement.prepared` will only record the last `sei.Prepared` for all statements, which is unnecessary. You can just keep the first `sei.Prepared` in `newStmtSummaryByDigestElement` and avoids updating `ssElement.prepared` every time.